bindings/2.204/GeometryDash.bro

```diff
@@ -6079,8 +6107,8 @@ class GJBaseGameLayer : cocos2d::CCLayer, TriggerEffectDelegate {
 	TodoReturn toggleGroup(int, bool);
 	TodoReturn toggleLockPlayer(bool, bool);
 	TodoReturn togglePlayerStreakBlend(bool);
-	void togglePlayerVisibility(bool, bool) = win 0x1B9320;
-	TodoReturn togglePlayerVisibility(bool); //inlined on windows
+	TodoReturn togglePlayerVisibility(bool, bool);
```

Comment:
why is this changed to TodoReturn and the address removed

Reply:
There I saw that the parameters for precisely that address come out with only one, and Ghidra shows it as one too. I thought they made a mistake. (about the TodoReturn, I'll fix it :c)